index.html

```diff
@@ -23,7 +23,9 @@
     <div id="welcome" class="show">
         <div class="heading">
             <h1>WEBEXTENSIONS</h1>
-            <button onclick="changeScreen('questions')" class="button">Let's create</button>
+            <button onclick="changeScreen('questions')" class="button" style="vertical-align:middle">Let's create for
```

Comment:
No inline style

Reply:
You can use `display: inline-flex;` with `align-items: center;` if you want to vertically align button content instead of `display: inline-block` that is being used rn.

Reply:
Thanks for the Input. Turns I didn't have to include that inline style at all :)